homeassistant/components/climate/__init__.py

```diff
@@ -43,6 +43,9 @@
 STATE_AUTO = "auto"
 STATE_DRY = "dry"
 STATE_FAN_ONLY = "fan_only"
+STATE_AWAY_HOLD = "away"
+STATE_HOME_HOLD = "home"
+STATE_TEMPERATURE_HOLD = "temp"
```

Comment:
@MartinHjelmare I think you are correct in being picky, because consistent and meaningful naming is one of the best mechanisms of helping to keep the code maintainable. I appreciate your input.